README.md

```diff
@@ -13,21 +13,59 @@ This package contains information for spinning up your Ghost Pinbot server used
 
 ## Start
 
-There are various ways of how to start Ghost Pinbot server. There is a `Dockerfile` which can be used to build the image and deploy it on your infrastructure.
+There are various ways of how to start the app depending on how you want to run the Ghost Pinbot application. The application supports 
+three different modes:
 
-The `docker-compose.yml` can be a good start to bootstrap Ghost Pinbot server:
+- `API`: service for communicating with the Ghost Pinbot API
+    
+- `PEER`: service for handling Ghost Thread functionalities 
 
-```
-$ docker-compose up
-```
+- `BUNDLED`: service which bundles both the API and PEER services.
 
-If you plan to deploy the Ghost Pinbot server on AWS there are Terraform scripts located in `deploy/aws-terraform-example`. 
+There is a `Dockerfile` which can be used to build the image and deploy it on your infrastructure.
+
+**Note**: the same `Dockerfile` is used for all the execution modes. This needs to be separated in some of the future versions of the Ghost Pinbot.
+
+#### Local
+
+The `docker-compose.yml` can be a good start to bootstrap Ghost Pinbot server.
+
+- `./docker-compose up`
+
+#### Heroku
+
+In order to run the Ghost Pinbot service on Heroku run the following script from the root dir.
+
+- `./deploy/heroku/build.sh <API_APP_NAME> <PEER_APP_NAME>`
+
+That script will create the script for deployment:
+
+- `./deploy/heroku/build/deploy.sh`
+
+Just run the script and your services will be deployed on Heroku. For example, if you run:
+
+- `./deploy/heroku/build.sh ghostpinbot-api ghostpinbot-api`
```

Comment:
last param here should be `*-peer` right?

Reply:
good catch! yep